sdata/README.md

```diff
@@ -0,0 +1,3 @@
+# Grafana dataplane sdata
+
```

Comment:
I think still "experimental" note perhaps? I want us to be free to really change signatures or even the approach until we get this is tried out in a few places, and that used to be in the path.